server/src/modules/playlists/dto/create-playlist.dto.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsNotEmpty,
+  IsObject,
+  IsOptional,
+  IsString,
+  MaxLength,
+  MinLength,
+  ValidateNested,
+} from 'class-validator';
+import { IdDto } from 'src/common/dto';
+
+export class CreatePlaylistDto {
+  @IsNotEmpty()
+  @IsString()
+  @MinLength(1)
+  @MaxLength(32)
+  @ApiProperty({
+    example: 'Music for study',
+    required: true,
+    nullable: false,
+    minLength: 1,
+    maxLength: 32,
+  })
+  public readonly name: string;
+
+  @IsOptional()
+  @IsObject()
+  @ValidateNested()
+  @ApiProperty({
+    type: IdDto,
+    required: false,
+    nullable: true,
+    example: { id: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx' },
+  })
+  public readonly user: IdDto;
```

Comment:
There is no need to take the user ID through the request body, we can get it directly from the auth token or session payload.
After implementing auth functionality, we will need to implement getting user id (and passing it for creating entity) in the controller, for now I suggest simply removing the user from the dto, since this field is nullable